packages/lit-dev-content/site/docs/components/decorators.md

```diff
@@ -48,7 +48,8 @@ See [Reactive properties](/docs/components/properties/) for more information abo
 | {% api "@query" "query" %} | Defines a property that returns an element in the component template. | [Shadow DOM](/docs/components/shadow-dom/#query) |
 | {% api "@queryAll" "queryAll" %} | Defines a property that returns a list of elements in the component template. | [Shadow DOM](/docs/components/shadow-dom/#query-all) |
 | {% api "@queryAsync" "queryAsync" %} | Defines a property that returns a promise that resolves to an element in the component template. | [Shadow DOM](/docs/components/shadow-dom/#query-async) |
-| {% api "@queryAssignedNodes" "queryAssignedNodes" %} | Defines a property that returns the children assigned to a specific slot. | [Shadow DOM](/docs/components/shadow-dom/#query-assigned-nodes) |
+| {% api "@queryAssignedElements" "queryAssignedNodes" %} | Defines a property that returns the child elements assigned to a specific slot. | [Shadow DOM](/docs/components/shadow-dom/#query-assigned-elements) |
```

Comment:
Until API is updated with Lit 2.1 generated API, linking to a non existent API crashes the build.

TODO: update with `queryAssignedElements`.

Reply:
Done!